packages/cactus-test-tooling/src/main/typescript/besu/besu-test-ledger.ts

```diff
@@ -11,12 +11,14 @@ export interface IBesuTestLedgerConstructorOptions {
   containerImageVersion?: string;
   containerImageName?: string;
   rpcApiHttpPort?: number;
+  cmdArgs?: string
```

Comment:
Could this be an array of strings instead of one string?
Makes it easier to deal with some exotic/special characters on a per argument basis if/when necessary.
Also recommend an array of strings instead of just one big string to keep with the conventions of the related software such as NodeJS child_process/spawn APIs and dockerode as well.

The default value can be an empty array and when calling docker.run(...) it's easy to concatenate the additional cmd args coming from the constructor and the ones prescribed by the logic of `BesuTestLedger` itself.